lib/double_entry/reporting/aggregate.rb

```diff
@@ -93,16 +112,17 @@ def range_is_complete?
 
       def field_hash
         {
-          :function => function,
-          :account => account,
-          :code => code,
-          :year => range.year,
-          :month => range.month,
-          :week => range.week,
-          :day => range.day,
-          :hour => range.hour,
-          :filter => filter.inspect,
-          :range_type => range.range_type.to_s,
+          :function        => function,
```

Comment:
There are some places in the code where we use `:key => value` and others `key: value`. Maybe we should make them consistent